### PR TITLE
First draft of functional tests

### PR DESCRIPTION
**Pull request by kucerakk**
Depends-On: https://github.com/opentelekomcloud-infra/otc-zuul-jobs/pull/32

### CODE REVIEW

**Review by gtema — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Review by gtema — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — DISMISSED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Comment by gtema** [comment]
recheck post

**Comment by gtema** [comment]
recheck post

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

File: roles/prepare-rancher-env/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+packages:
+  - pytest
+  - selenium
+  - docker
+
+rancher_prefix: rancher-
+
+rancher_password: '{{ rancher_credentials.password }}'
+rancher_driver_location: https://github.com/opentelekomcloud/kontainer-engine-driver-otc/releases/download/v0.2.1/kontainer-engine-driver-otc-0.2.1-linux-amd64.tgz
```
Comment (gtema) [inline roles/prepare-rancher-env/defaults/main.yml:14]: let's make this a variable coming from job (default points here, but job should be able to override it)

**Comment by outcatcher** [comment]
I don't think, that this is a right location for the func tests. They should be triggered for PRs in both driver and UI. Besides they really blow this repo up.

**Review by outcatcher — CHANGES_REQUESTED** [review]
I don't think, that this is a right locationto keep func tests. They should be triggered for PRs in both driver and UI. Besides they really blow this repo up.

**Review by vladimirhasko — DISMISSED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — DISMISSED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — DISMISSED** [review]

**Review by vladimirhasko — APPROVED** [review]

**Review by vladimirhasko — DISMISSED** [review]

**Comment by gtema** [comment]
recheck

**Comment by gtema** [comment]
recheck

**Review by gtema — DISMISSED** [review]
trigger post-review

**Comment by gtema** [comment]
recheck

**Comment by outcatcher** [comment]
Done in https://github.com/opentelekomcloud-infra/rancher-integration-tests

**Pull request closed without merging**